### PR TITLE
Now rely on .gnarrc

### PR DESCRIPTION
**Pull request by alxjrvs**
Finish the loop: Use `.gnarrc` to boot rails 7. 

This begs the question: What is `gnarrails`? In this model, the whole app essentially exists to make sure that the `.railsrc` gets properly downloaded, then removed. 

I think with `.gnarrc`, it would make more sense to consider closing this and beefing up `gnar-cli` to include `rails new` functonality. But all of this is just fanfiction for the time being, so who can say?

### CODE REVIEW

File: lib/gnarails/cli/application.rb [hunk 1]
```diff
@@ -13,50 +14,18 @@ class Application < Thor
...
-        You should also be able to pass any other arguments you would expect to
-        be able to when generating a new rails app. Use `rails -h` for more
-        information.
+        Learn more about our Rails Configuration here: https://github.com/TheGnarCo/.gnarrc
       LONGDESC
+
       def new(name)
+        create_file ".railsrc", Net::HTTP.get(URI("https://raw.githubusercontent.com/TheGnarCo/.gnarrc/main/rails/7/.railsrc"))
         Kernel.system command(name, options)
+        remove_file ".railsrc"
```
Comment (efertsch) [inline lib/gnarails/cli/application.rb:23]: Why do we need to remove this file?

File: lib/gnarails/cli/application.rb [hunk 2]
```diff
@@ -13,50 +14,18 @@ class Application < Thor
...
-        def cli_option(key, value)
-          case value
-          when false
-            ""
-          when true
-            " --#{key}"
-          else
-            " --#{key}=#{value}"
-          end
+          "rails new #{name} --rc=.railsrc"
```
Comment (efertsch) [inline lib/gnarails/cli/application.rb:28]: Thoughts on making `--rc=.railsrc` a constant?

**Review by efertsch — APPROVED** [review]
This looks good to me! Might be worth involving @mikestone14 and @ngmaloney (who I will annoyingly tag in all of these reviews) for visibility 😄

File: lib/gnarails/cli/application.rb (hunk 1 above)
Comment (alxjrvs) [reply lib/gnarails/cli/application.rb:23]: it feels weird to leave this behind. If I was going to take another pass at this, I'd probably not even download the file and just use ruby to get the raw contents and append them to the `rails new` task manually.

File: lib/gnarails/cli/application.rb (hunk 2 above)
Comment (alxjrvs) [reply lib/gnarails/cli/application.rb:28]: favorable!

**Pull request merged**